scripts/build/TestPlatform.Settings.targets

```diff
@@ -53,7 +53,14 @@
   <ItemGroup>
     <PackageReference Condition=" '$(OS)' != 'Windows_NT' " Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" PrivateAssets="All" />
   </ItemGroup>
-  
+
+  <ItemGroup Condition=" '$(UseBannedApiAnalyzers)' == 'true' ">
```

Comment:
Any preference between copy-pasting the full package reference block in all projects using the banned api analyzer or this (i.e. adding a simple property)?